src/App.css

```diff
@@ -0,0 +1,19 @@
+html,
```

Comment:
Great use of CSS, great minimalistic feel. Displays all relevant information clearly.